.travis.yml

```diff
@@ -31,16 +30,19 @@ install:
                :prompt nil)'
 
 script:
-  - ros -e '(ql:quickload :emotiq-test) 
-            (when (asdf:test-system :emotiq)
-               (uiop:quit 0))'
-  - ros -e '(ql:quickload :cosi-test)
-            (when (asdf:test-system :cosi)
-               (uiop:quit 0))'
-  # Speculatively hook into Coveralls <https://coveralls.io/>
-  # Coveralls invocation (currently unused)
-  #                 
-  # - ros -s prove -s cl-coveralls
-  #       -e '(or (coveralls:with-coveralls (:exclude (list "t"))
-  #                 (prove:run :emotiq-prove))
-  #               (uiop:quit -1))'
+  - ros -e "(loop
+              :for i :upfrom 0
```

Comment:
how about upfrom 1

Reply:
Because only FORTRAN programmers know what `option base` does.

And `(ash -1 1)` is `-2` whereas `(ash -1 0)` is `1`

Reply:
I don't get it. Do I have to know a lot about Fortran to understand that? Oh well, I suggested it to go with a change below, but I'll "fix" it below...